opentracing-tracerresolver/pom.xml

```diff
@@ -38,6 +38,7 @@
         <dependency>
             <groupId>io.opentracing</groupId>
             <artifactId>opentracing-api</artifactId>
+            <scope>provided</scope>
```

Comment:
Except that `opentracing-api` is _not_ an optional dependency but an actual runtime-required dependency.